Modules/FT0/include/FT0/ChannelsCheck.h

```diff
@@ -0,0 +1,44 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   ChannelsCheck.h
+/// \author Milosz Filus
+/// Dummy check for FT0 detector
```

Comment:
Is it really a dummy check ?

Reply:
Hello,
all of these checks will be probably changed by experts who know what to check in this data. That is why I called them dummy checks.
The same case for monitoring objects.

The post processing histogram generation is just to show how to read tree from database and then do some stuff on that tree.